metaflow/package.py

```diff
@@ -16,7 +17,7 @@
     import io
     BytesIO = io.BytesIO
 
-DEFAULT_SUFFIXES = ['.py']
+DEFAULT_SUFFIXES = ['.py', '.R']
```

Comment:
Do we want to include `.Rmd` by default as well? The flow can be launched from R markdown files.

Reply:
Good idea! I will add that in.